apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -63,10 +63,7 @@ h2 + .c-card-wrapper {
     a:before {
         content: '';
         position: absolute;
-        top: 0;
-        right: 0;
-        bottom: 0;
-        left: 0;
+        inset: 0;
```

Comment:
are you happy with the [browser support](https://caniuse.com/?search=inset) for `inset`?

Reply:
This is very cool – didn't know about this property